packages/driver/cypress/integration/issues/7088_spec.js

```diff
@@ -0,0 +1,20 @@
+/// <reference types="cypress" />
+
+describe('slatejs', () => {
+  it('test', () => {
+    cy.visit('fixtures/issue-7088.html')
+
+    const testString = 'Hello world'
+
+    cy.get('[data-slate-editor="true"]')
+    .type(testString, { noUpdate: true })
+
+    cy.contains('[data-slate-string="true"]', testString)
+    .should('be.visible')
+
+    cy.get('[data-slate-editor="true"]')
+    .type('{ctrl}{shift}{backspace}', { release: false, noUpdate: true })
```

Comment:
I tried removing the noUpdate options here and the test still passed, is there a test case that fails w/o the option?

Reply:
You're right. `noUpdate` isn't necessary. 

I added it because there were some cases when `cy.type()` breaks the html structure inside slatejs. It seems that it was because `beforeinput` part was buggy.

Reply:
the reason you previously needed noUpdate was due to the event not being set as cancelable https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/beforeinput_event, fixed now so we will respect a cancelled event and not perform the default action